build-scripts/set-env-variables.sh

```diff
@@ -49,6 +49,10 @@ export K8S_DQLITE_TAG="${K8S_DQLITE_TAG:-v1.0.3}"
 
 export KUBE_SNAP_ROOT="$(readlink -f .)"
 
+export ADDONS_REPO="${ADDONS_REPO:-https://github.com/canonical/microk8s-addons}"
+export ADDONS_REPO_BRANCH="${ADDONS_REPO_BRANCH:-feature/initial-experimentation}"
```

Comment:
This has to change right after the https://github.com/canonical/microk8s-addons/pull/1 is merged so we get the addons from the main branch.